en/debug/sensor_uorb_topic_debugging.md

```diff
@@ -0,0 +1,17 @@
+# Sensor/Topic Debugging using the Listener Command
+
+The uORB is an asynchronous `publish()` / `subscribe()` messaging API used for
+inter-thread/inter-process communication. The `listener` command can be used from the *QGroundControl MAVLink Console* to inspect topic (message) values, including the current values published by sensors.
```

Comment:
@bkueng The text is deliberately designed this way (my original version was just as you have suggested!) The reason it exactly as you say - it is QGC that is interesting here - not the shell.

It is possible it is pushed "too far" down in which case this note could be in the set with the other ones.

Does that make sense?

As an aside, it would be great to talk more about debugging using QGC, but I know little about that yet. I have created https://github.com/mavlink/qgc-user-guide/issues/71 so we will eventually add more information in QGC on this topic.